### DISCUSSION TITLE
I can´t add Data Base in SQLiteStudio

### DISCUSSION THREAD

**Discussion by golivencia** [pawelsalawa/sqlitestudio#4867] [Q&A]
### Details
[17:50:13] Could not add database C:\Users\Gustavo\Documents\Gustavo\Testing\Curso Udemy\Bases de Datos y SQL\Pasoapaso.db to list.

### Operating system
Windows 10 Enterprisse LTSC

### SQLiteStudio version
SQLiteStudio 3.4.4
![error](https://github.com/pawelsalawa/sqlitestudio/assets/144492871/cc851516-2c5a-4038-9521-bbf8883c3c22)

**Comment by SamHobbsOrg** [comment]
Are you sure the path is valid? Can you go to `C:\Users\Gustavo\Documents\Gustavo\Testing\Curso Udemy\Bases de Datos y SQL` in the files explorer?

**Comment by pawelsalawa** [comment]
Please run SQLiteStudio in debug mode - run command line (`cmd`) and there go to the folder where your SQLiteStudio is, for example:
```
cd "C:\Program Files\SQLiteStudio"
```
and run it from there using `-d` option:
```
SQLiteStudio.exe -d
```

Then after the application starts hit the `F12` key. A debug console will appear to you (if not, you can use `Help`->`Show Debug Console` menu entry).

Then go back to the main window and try to add your database again. When you get the "Could not add database" message, go to the Debug Console window and check what is the detailed error there.
